common_voice_CV/vademecum_CV.md

```diff
@@ -0,0 +1,64 @@
+**_Si consiglia di leggere prima il file README.md_**
+
+# Common Voice
+
+## mozilla
+
+
+
+## Che cos’è Common Voice?
+
+Common Voice è parte di un’iniziativa Mozilla per insegnare alle 
+macchine come la gente parla nella vita reale. In aggiunta al dataset di
+Common Voice, stiamo costruendo un motore di riconoscimento vocale open
+source chiamato Deep Speech.
+Entrambi i progetti rientrano nella nostra missione 
+di colmare il divario nel settore del riconoscimento vocale digitale. Le
+tecnologie di riconoscimento vocale conferiscono una dimensione umana 
+ai nostri dispositivi, ma gli sviluppatori hanno bisogno di un’enorme 
+quantità di dati vocali per crearle. Attualmente, la maggior parte di 
+questi dati è costosa e proprietaria. Vogliamo rendere i dati vocali 
+disponibili liberamente e pubblicamente e assicurarci che questi dati 
+rispecchino la diversità delle persone reali. Insieme possiamo rendere 
+il riconoscimento vocale migliore per tutti.
+
+## Come contribuire al progetto?
+
+### Vai su https://voice.mozilla.org/it                     
+
+<img src="file:///C:/Users/saver/Documents/GitHub/firefox-vademecum/images/qrcodes/commonvoice.png" alt="img" style="zoom:20%;" />
+
+### Scegli che cosa vuoi fare                     
+
+Puoi scegliere di **registrare** la tua voce o **convalidare** le registrazioni immesse da altri.
+
+<img src="file:///C:/Users/saver/Documents/GitHub/firefox-vademecum/images/ascolta_cv.png" alt="img" style="zoom:20%;" /> <img src="file:///C:/Users/saver/Documents/GitHub/firefox-vademecum/images/parla_cv.png" alt="img" style="zoom:20%;" />
+
+## Perché creare un account?
```

Comment:
usiamo account o profilo sul sito?

Reply:
Profilo, ma il problema è si accede con un "account Firefox" (tradotto così in italiano) ...

Reply:
Io accetterei questa versione e aprire un issue, nel quale segnalare tutte le modifiche da fare. Penso sia più facile, in quel modo, no?

Reply:
Ho messo "profilo" dove dice "eliminare profilo" ma ho lasciato "account Firefox"